be/issue-tracker/src/test/java/team08/issuetracker/label/service/LabelServiceTest.java

```diff
@@ -0,0 +1,59 @@
+package team08.issuetracker.label.service;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import team08.issuetracker.exception.label.LabelNotFoundException;
+import team08.issuetracker.label.model.Label;
+import team08.issuetracker.label.model.dto.LabelCreationRequest;
+import team08.issuetracker.label.repository.LabelRepository;
+
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+@SpringBootTest
+class LabelServiceTest {
+
+    @Autowired
+    private LabelService labelService;
+
+    @MockBean
+    private LabelRepository labelRepository;
+
+    @Test
+    void testCreateLabel() {
+        LabelCreationRequest labelCreationRequest = new LabelCreationRequest("New Label", "New Description", "#FFFFFF", true);
+        Label expectedLabel = new Label("New Label", "New Description", "#FFFFFF", true);
+
+        when(labelRepository.save(any(Label.class))).thenReturn(expectedLabel);
+
+        Label createdLabel = labelService.createLabel(labelCreationRequest);
+
+        assertThat(createdLabel.getName()).isEqualTo("New Label");
```

Comment:
```suggestion
        assertThat(createdLabel.getName())
                .as("라벨이 잘 생성 되어야 한다")
                .isEqualTo("New Label");
```

어떤 의도로 테스트코드를 작성한건지 명시해주면 훨씬 더 좋을꺼같아요!